Modules/IO/GDCM/include/itkGDCMImageIO.h

```diff
@@ -53,6 +53,9 @@ namespace itk
  *
  * GDCM build, instead of the one included within ITK itself.
  *
+ * The compressors supported include "JPEG2000" (default), and
```

Comment:
At least in my fork, I can do JPEG 2000/JPEG/JPEG-LS/RLE. So I need to do my homework here, first.

Reply:
There are quite a few definitions related to JPEG [here](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/ThirdParty/GDCM/src/gdcm/Source/DataDictionary/gdcmUIDs.h#L349-L376), it's not clear which are supported compressors. We can save exposing additional one for another patch. This documentation is for the features supported by the `ImageIO` not the library. Perhaps it should indicate that these are the "compressors" supported for writing.